Assignment/solution_student/index.html

```diff
@@ -10,24 +9,49 @@
 
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
+
+<header>
```

Comment:
Oops! Looks like you accidentally placed your page's `<header>` element _outside_ of the `<body>` element, but remember, **everything** you see on the page goes _inside_ the `<body>` element. So just make sure you move this down inside of the opening `<body>` element on line `21`. Refer to the assignment feedback I gave about this very topic if it helps: [https://github.com/GA-Chicago-FEWD6/Week_02_Styling/blob/master/assignment-feedback.md#head-or-header](https://github.com/GA-Chicago-FEWD6/Week_02_Styling/blob/master/assignment-feedback.md#head-or-header)